contracts/vested-escrow/VeERC721Extension.sol

```diff
@@ -24,8 +26,7 @@ contract VeERC721Extension is SuAccessControlAuthenticated, VotesUpgradeable {
 
     function initialize(address _accessControlSingleton, address _nftToken, address _bonus) public initializer
     {
-        //        __ERC721_init(string.concat("vested escrow ", ERC721(_nftToken).name()), string.concat("ve", ERC721(_nftToken).symbol()));
-        __SuAuthenticated_init(_accessControlSingleton);
+        __SuVoteToken__init(_accessControlSingleton, "VeERC721Extension");
```

Comment:
why do we pass same string every time, instead of string.concat("ve", ERC721(_nftToken).symbol()?